index.js

```diff
@@ -49,15 +66,15 @@ module.exports = function (text, opts) {
 
 	var contentWidth = widestLine(text) + padding.left + padding.right;
 	var horizontal = repeating('─', contentWidth);
-	var top = colorizeBorder('┌' + horizontal + '┐');
-	var bottom = colorizeBorder('└' + horizontal + '┘');
+	var top = colorizeBorder(repeating('\n', margin.top) + repeating(' ', margin.left) + '┌' + horizontal + '┐' + repeating(' ', margin.right));
+	var bottom = colorizeBorder(repeating(' ', margin.left) + '└' + horizontal + '┘' + repeating('\n', margin.bottom) + repeating(' ', margin.right));
 	var side = colorizeBorder('│');
 
 	var middle = lines.map(function (line) {
 		var paddingLeft = repeating(' ', padding.left);
 		var paddingRight = repeating(' ', contentWidth - stringWidth(line) - padding.left);
 
-		return side + paddingLeft + line + paddingRight + side;
+		return repeating(' ', margin.left) + side + paddingLeft + line + paddingRight + side + repeating(' ', margin.right);
```

Comment:
I would drop adding the actual right margin as it just makes it break lines earlier on narrow terminal windows. It has no effect anyways. Should still be handled in the options handling, but not added to the output.